modules/home/assets/js/components/home-screen.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+import { Box } from '@elementor/ui';
+import TopSection from './top-section';
+
+const HomeScreen = () => {
+	const baseUrl = elementorAppConfig.pages_url;
+	const [ topScreenProps ] = useState( { videoUrl: 'https://elementor.com/academy/getting-started-with-elementor/', ctaUrl: baseUrl, embedUrl: 'https://www.youtube.com/embed/icTcREd1tAg?si=MPamCEWNeRR_VdAn&amp;controls=0' } );
```

Comment:
Maybe use multiple lines for readability.

```suggestion
	const [ topScreenProps ] = useState( {
	videoUrl: 'https://elementor.com/academy/getting-started-with-elementor/',
	ctaUrl: baseUrl,
	embedUrl: 'https://www.youtube.com/embed/icTcREd1tAg?si=MPamCEWNeRR_VdAn&amp;controls=0'
	} );
```